packages/core/src/lib/utils/address.ts

```diff
@@ -0,0 +1,66 @@
+import { fromBech32 } from '@cosmjs/encoding';
+import CryptoJS from 'crypto-js';
+import { ec as EllipticCurve } from 'elliptic';
+
+export const isValidAddress = (address: string) => {
```

Comment:
this currently validates if it's a valid bech32 address, but i think it should validate sei addresses only by making sure `prefix === 'sei'`